javascript/index.md

```diff
@@ -23,28 +23,31 @@ JavaScript tasks.
 
 Bring a computer with a web browser and text editor installed. We recommend
 [Chrome](https://www.google.com/chrome/browser/desktop/index.html) and
-[Sublime Text](http://www.sublimetext.com/).
+[Sublime Text](http://www.sublimetext.com/) or [Atom](https://atom.io/).
 
 This talk assumes you've programmed before, either for a class or on your own
 time.
 
 
 ## Resources
 
-This workshop has been run before, and these are the resources from last time.
+Here are the resources we are going to be using as well as some resources from
+previous years.
 If you attend this workshop, we'll work through all of these resources to help
 you complete a fun JavaScript project!
 
-1. [Introduction to JavaScript]({{ site.baseurl }}/javascript/js-intro/)
-1. [JavaScript Lab]({{ site.baseurl }}/javascript/lab/)
+1. [Introduction to JavaScript Slides](https://wafflecohn.github.io/javascript-lab/intro-javascript-slides.pdf)
 1. [HTML & CSS Recap]({{ site.baseurl }}/javascript/html-css/)
-1. [Demo of Final Project]({{ site.baseurl }}/javascript/demo/)
-1. [Source code of Final Project](https://github.com/jez/jquery-lab/)
+1. [JavaScript Lab]({{ site.baseurl }}/javascript/lab/)
+1. [Demo of Final Project](https://wafflecohn.github.io/javascript-lab/)
+1. [Source code of Final Project](https://github.com/WaffleCohn/javascript-lab)
+1. [Old Introduction to JavaScript]({{ site.baseurl }}/javascript/js-intro/)
+1. [Old JavaScript Lab]({{ site.baseurl }}/javascript/lab/oldlab.html)
```

Comment:
@skrulcik this should just be `{{ site.baseurl }}/javascript/lab/oldlab`, right?

Reply:
Well the way this is structured now, `.html` is the way to go. But most of the site would move `oldlab.md` to `oldlab/index.md`, which allows us to drop the filename extension like you describe.

To stay consistent across the repo, Ari can you move to the `index.md` structure so we can keep all links extensionless?

Reply:
Sure